docs/guides/image-update.md

```diff
@@ -949,4 +942,5 @@ Update the `ImageRepository.spec.secretRef` to point to it.
 
 It is also possible to create [Repository Scoped Tokens](https://docs.microsoft.com/en-us/azure/container-registry/container-registry-repository-scoped-permissions).
 
-!!! Note that this feature is in preview and does have limitations.
+!!! warning
+    Repository Scoped Tokens are in preview and do have limitations.
```

Comment:
Are these limitations documented somewhere so that we can link to them?

Reply:
Yep, from the same doc as the line above:
https://docs.microsoft.com/en-us/azure/container-registry/container-registry-repository-scoped-permissions#preview-limitations

There are a lot of these Preview features that we're point people to use with Azure.
I've included a lot of these style of warnings in our own docs, but they are all over the Azure ones as well.

It adds some maintenance burden, but there are so many pieces that people have to keep in their head when putting together the integration, that I figured it would be good value add to have the warnings on our side as well.